README.md

```diff
@@ -207,9 +209,9 @@ For information about the governance of the Node.js project, see
 
 <details>
 
-<summary>Emeriti</summary>
+<summary>Regular members</summary>
 
-### TSC emeriti
+#### TSC regular members
```

Comment:
Do we still need an emeriti section as suggested in https://github.com/nodejs/TSC/pull/1350#issue-1612674897?

> * Leave TSC emeriti that are not current collaborators as emeriti.

Perhaps leaving that for a follow-up was intentional? - if so please ignore.

Reply:
Oh, right, I was supposed to split it into two lists. Thanks. I'll do that!

Reply:
Ok, all fixed. Thanks for catching that!

Reply:
Thanks! Should we also reach out to those who are going to be added back as TSC Members? Unsure if it's likely, but could some prefer to explicitly remain emeritus?

Reply:
I will definitely let each one know. I 100% expect some folks to ask to be moved to Emeritus.

Reply:
I thought we had said that all existing emeritus members would stay that way?